src/themes/mat/variables.js

```diff
@@ -5,9 +5,9 @@ export default {
 
 	headingSmallSize: '1.6rem',
 	headingSmallLineHeight: '2.3rem',
-	headingMediumSize: '1.8rem',
+	headingRegularSize: '1.8rem',
```

Comment:
Keep in mind that when you change a font-size, you might want to have a look at the corresponding line-height as well.